.github/workflows/semver_checks.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       id: semver-pr-check
       run: |
         set +e
-        cargo semver-checks -p scylla -p scylla-cql --baseline-rev "$PR_BASE"
+        make semver-rev rev="$PR_BASE"
```

Comment:
Ok, I see that you replace the manual calls to semver-checks in the last commit. Please move the commit that introduces the script to the beginning.